### DISCUSSION TITLE
Background color is lost in striped tables

### DISCUSSION THREAD

**Discussion by RafaelSLopes** [twbs/bootstrap#35370] [General]
The background color (bg-success and others) of cells in striped tables (table-striped) together with the table color (table-info and others) are lost in the striped rows.. In Bootstrap v4 it worked perfectly. Any solution?

Exemple: 
(In the example only cell "Jacob" will receive the background color, while cell "Mark" will receive the background color of the striped table.)
```
<table class="table table-sm table-striped table-info">
  <thead>
    <tr>
      <th scope="col">#</th>
      <th scope="col">First</th>
      <th scope="col">Last</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">1</th>
      <td class="bg-success">Mark</td>
      <td>Otto</td>
    </tr>
    <tr>
      <th scope="row">2</th>
      <td class="bg-warning">Jacob</td>
      <td>Thornton</td>
    </tr>
  </tbody>
</table>
```

**Comment by DanielBadura** [comment]
Having the same problem with striped table but only with `bg-danger` e.g. `bg-success` and `bg-warning` is working for me. Using Bootstrap `5.1.3`. The box shadow that is added for the stripe effekt is causing this issue.
